spec/view_objects/funding/trainee_summary_view_spec.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+module Funding
+  describe TraineeSummaryView do
+    let(:maths_scholarship_amount) { 500000 }
+    let(:maths_bursary_amount) { 200000 }
+    let(:maths_grant_amount) { 300000 }
+    let(:maths_scholarship_trainees) { 3 }
+    let(:maths_bursary_trainees) { 2 }
+    let(:maths_grant_trainees) { 1 }
```

Comment:
```suggestion
    let(:maths_grant_trainees) { 3 }
```

Using a number greater than one gives us more confidence that the underlining calculation uses multiplication.